src/karmen_frontend/src/components/tabs/printer/__tests__/control-tabs.js

```diff
@@ -0,0 +1,141 @@
+import React from "react";
+import ControlsTab from "../controls-tab";
+import { render, waitFor } from "@testing-library/react";
+
+require("jest-fetch-mock").enableMocks();
+
+const createPrinter = (accessLevel, connected, state) => {
```

Comment:
This is huge testing data, does'n it make sense to share it with other tests?

Reply:
Currently we don't have any other tests.